### PR TITLE
Delete issue

### PR DESCRIPTION
**Pull request by DanGTZ**
Added tests for #825

### CODE REVIEW

**Comment by flovilmart** [comment]
@DanGTZ you're starting to get how it's done! That's really appreciated you open those PR's.

**Comment by DanGTZ** [comment]
@flovilmart I can't figure out why testFailingSave() is successful in Android Studio when I launch ParseObjectTest tests, but fails when launching all tests via gradle command (as you can see on Travis).
I would like to keep this non-regression test so I will add a commit to make it work, but not sure it's the right way to fix this.

**Comment by flovilmart** [comment]
Perhaps a side effect between the tesTs, state that is leaking from the previous one or something like that?

**Comment by DanGTZ** [comment]
Yes it seems to be a side effect between test classes.

**Comment by flovilmart** [comment]
Awesome! I’ll let @jawnypoo and @rogerhu have a look at the code!

**Comment by DanGTZ** [comment]
@Jawnnypoo why did the merge failed ?

File: Parse/src/test/java/com/parse/NetworkObjectControllerTest.java [hunk 1]
```diff
@@ -40,6 +42,9 @@
 @Config(constants = BuildConfig.class, sdk = TestHelper.ROBOLECTRIC_SDK_VERSION)
 public class NetworkObjectControllerTest {
 
+  @Rule
+  public ExpectedException thrown= ExpectedException.none();
```
Comment (rogerhu) [inline Parse/src/test/java/com/parse/NetworkObjectControllerTest.java]: can you add a space?

**Comment by rogerhu** [comment]
Please rebase. :)

**Comment by DanGTZ** [comment]
@rogerhu I did a change on my master branch : https://github.com/parse-community/Parse-SDK-Android/pull/837
Is it OK to rebase this change into this branch?

**Comment by DanGTZ** [comment]
@rogerhu is all ok now ?

**Comment by Jawnnypoo** [comment]
Thanks @DanGTZ !

**Pull request merged**